_data/glossary/statefulset.yml

```diff
@@ -0,0 +1,18 @@
+id: statefulset
+name: StatefulSet
+formerly:
+- PetSet
+related:
+- Deployment
+- Pod
+tags:
+- Storage
+- Workload
+- API Object
+short-description: |
+  Manages the deployment and scaling of a set of replica Pods, *and provides guarantees about ordering*. It does so by maintaining a *unique*, sticky identity for each of its Pods.
```

Comment:
I believe the kind in the YAML config file is `StatefulSet`, so maybe we should change that page to match the singular form. In general though, could we establish a style guideline to use the singular form for feature and object names?